frontend/src/lib/components/Milestone.svelte

```diff
@@ -11,12 +11,18 @@
     let autoGoToNextMilestone: boolean = false;
 
     // build list of possible image urls at build time to be able to dynamically use them at run time:
-    const images: Record<string, string> = import.meta.glob('$lib/assets/*.jpg', {
+    const img_urls: Record<string, string> = import.meta.glob('$lib/assets/*.jpg', {
         eager: true,
         query: '?url',
         import: 'default'
     });
 
+    let currentImageIndex:number = 0;
+    const imageInterval = 5000;
```

Comment:
is 5s perhaps a little long as a delay between images? I'm not sure where  a good delay would be that makes people realize that there is content to look at here before they get impatient and start doing things....

Reply:
Yes I'll make an issue for this - it's currently a bit long if you're just clicking an answer, but if you open the dropdown help text and try to read it then (at least for me) I found it a bit annoying when it kept updating - could also just cycle once then stop I guess.

Reply:
https://github.com/ssciwr/mondey-frontend-prototype/issues/62